packages/components/header/src/Header.tsx

```diff
@@ -82,32 +79,34 @@ function _Header<E extends ElementType = typeof HEADER_DEFAULT_TAG>(
     metadata,
     title,
     withBackButton,
-    variant = 'breadcrumb',
```

Comment:
i removed the variant prop as it was not obvious that variant had to be set to "title" specifically when there where no breadcrumbs or back-button.
I believe this logic is better handled inside the component. But will calrify with Daniel wether or not the size of the title should be forcable from outside